ayekan/opensearch/ayekan-cluster.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: opensearch.opster.io/v1
+kind: OpenSearchCluster
+metadata:
+  finalizers:
+  - Opster
+  name: ayekan-opensearch
+spec:
+  bootstrap:
+    resources: {}
+  confMgmt: {}
+  dashboards:
+    enable: true
+    opensearchCredentialsSecret: {}
+    replicas: 2
+    resources:
+      limits:
+        cpu: 200m
+        memory: 512Mi
+      requests:
+        cpu: 200m
+        memory: 512Mi
+    service:
+      type: ClusterIP
+    version: 2.3.0
+  general:
+    httpPort: 9200
+    monitoring: {}
+    serviceName: ayekan-opensearch
+    setVMMaxMapCount: true
+    version: 2.3.0
+  initHelper:
+    resources: {}
+  nodePools:
+  - component: nodes
+    diskSize: 100Gi
+    replicas: 3
+    resources:
+      limits:
+        cpu: "1"
+        memory: 4Gi
+      requests:
+        cpu: "1"
+        memory: 4Gi
+    roles:
+    - cluster_manager
+    - data
+---
+apiVersion: opensearch.opster.io/v1
+kind: OpensearchUser
+metadata:
+  name: fluentbit
+spec:
+  opensearchCluster:
+    name: ayekan-opensearch
+  passwordFrom:
+    name: fluentbit-password
+    key: password
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: fluentbit-password
+data:
+  password: cnViaW5vYnMK
```

Comment:
Obviously, we wouldn't want to merge this with the secret inlined.

Reply:
In the current version this has been migrated to secrets and the secret operator.